dune-project

```diff
@@ -21,7 +21,9 @@
   (ocaml (>= 4.08.0))
   (cohttp-lwt-unix (>= 2.5.4))
   (dune (>= 2.5.0))
+  dune-build-info
   (ppx_deriving (>= 4.5))
+  ppx_deriving_yojson
```

Comment:
It seems all the other deps are also accompanied by a version constraint. Can you add in, please? :)

Reply:
Good point! Version constraints are present now :)